packages/appeals-service-api/src/routes/v2/appeal-cases/_caseReference/lpa-questionnaire-submission/spec.yaml

```diff
@@ -84,3 +84,25 @@ paths:
             application/json:
               schema:
                  $ref: '#/components/schemas/ErrorBody'
+  /api/v2/appeal-cases/{caseReference}/lpa-questionnaire-submission/submit:
```

Comment:
Would there be opposition to dropping `submission` from the `lpa-questionnaire-submission`?